<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceDialog</class>
 <widget class="QWidget" name="ReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Replace</string>
  </property>
  <property name="windowIcon">
   <iconset resource="TextEditor.qrc">
    <normaloff>:/TextEditor/Resources/replace.svg</normaloff>:/TextEditor/Resources/replace.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Replace what:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editReplaceTo"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="checkCaseSensitive">
       <property name="text">
        <string>Case sensitive</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Replace to:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editReplaceWhat"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="buttonReplace">
       <property name="text">
        <string>Replace all</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TextEditor.qrc"/>
 </resources>
 <connections/>
</ui>
